### DISCUSSION TITLE
Updates to the video play button

### DISCUSSION THREAD

**Discussion by gladys-ng** [Shopify/polaris-react#6859] [Ideas] [answered]
### Proposed solution

Would it be possible to update the video play button to bring it up to date with the current design system while also providing more detailed information to merchants? The current play button is large and in some cases it impacts the visual composition of the thumbnail (see before/after below where the individual in the thumbnail is covered by the play button).

This work comes from https://github.com/Shopify/shopify/issues/355172 (https://github.com/Shopify/shopify/issues/355172) on the Admin Quality crew project board.

**Feature details:**

- Instead of a large circular play button, the play button will be located on the bottom left of the thumbnail and display the video duration
- There is a mobile and desktop version (which differ only in text styles, `mobile/button` vs `desktop/button`)
- Hit zone for the video would still be the entire thumbnail and not just the play button

### Rationale
The existing video play button hasn't been updated in awhile and looks out of place compared to the rest of the design system. It takes up a large portion of the thumbnail and in some cases can cover important visuals. The new design keeps the play button contained on the bottom left of the thumbnail and provides useful metadata about the video (i.e. duration) to merchants.

[Link to Figma](https://www.figma.com/file/QHr9O9CAn4su9j6jrdhhRU/Quick-wins?node-id=1463%3A25952)

![image](https://user-images.githubusercontent.com/50415277/183091277-823d738f-08b4-43da-aac1-046b108c8e41.png)

**Comment by yesenia-perezcruz** [comment, marked as answer]
@gladys-ng The proposed design looks much better. Great that the play button isn't covering people's faces. I don't have additional feedback, so I'd recommend your team submits this change as a pull request.

**Reply by alex-page** [reply]
New issue here: https://github.com/Shopify/polaris/issues/6864

**Comment by alex-page** [comment]
@gladys-ng @yesenia-perezcruz I am going to convert this discussion to an issue. @gladys-ng if there is a developer on your team that wants to update the Polaris component we would be happy to review it.
